brage-import/src/main/java/no/sikt/nva/brage/migration/record/license/NvaLicenseIdentifier.java

```diff
@@ -1,26 +0,0 @@
-package no.sikt.nva.brage.migration.record.license;
-
-import com.fasterxml.jackson.annotation.JsonValue;
-import nva.commons.core.JacocoGenerated;
-
-public enum NvaLicenseIdentifier {
```

Comment:
Igjen for at endringene her skal fungere må input json også være endre:
sample json: https://s3.console.aws.amazon.com/s3/object/brage-migration-input-files-750639270376?region=eu-west-1&prefix=krus/191027/1/195255.json

nva sandbox